app/public/templates/scene_nav_mesh.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
Where is this one used? Is it a sample?

Reply:
Yeah it is for creating the nav-mesh for a scene. For example, if we have a new scene, just change the render template here https://github.com/uwblueprint/cdc-backend/blob/master/app/routes/ui/scenario.py#L54 to "scene_nav_mesh.html" 😄 This will be useful for creating the nav-mesh objects once we have the actual scenes from designers

Reply:
Got it, great!